public/styles.css

```diff
@@ -117,3 +118,45 @@ input:focus {
   width: 100%;
   margin-top: 0;
 }
+
+.practice-card-container {
+  position: absolute;
```

Comment:
There has got to be a simpler way to lay this out than manually positioning everything. What are the limitations of Bootstrap's grid here? For horizontal centering, `margin: 0 auto;` is great. Plus, I believe there's a built-in Bootstrap helper class for it.